includes/class-wc-tracker.php

```diff
@@ -540,6 +541,49 @@ private static function get_admin_user_agents() {
 		return array_filter( (array) get_option( 'woocommerce_tracker_ua', array() ) );
 	}
 
+	/**
+	 * Get order counts by payment method
+	 *
+	 * @return array
+	 */
+	public static function get_order_counts_by_payment_method() {
+		global $wpdb;
+
+		$orders_by_payment_method = $wpdb->get_results(
+			"
+			SELECT
+			    order_gateway.payment_method AS payment_method,
+			    COUNT( id ) AS orders_count
+			FROM (
+				SELECT
+					orders.ID AS id,
+				    MAX(
+				        CASE
+				            WHEN meta_key = '_payment_method'
+				            THEN meta_value
+				        END
+				    ) AS payment_method
+				FROM {$wpdb->prefix}posts AS orders
+				LEFT JOIN {$wpdb->prefix}postmeta AS order_meta ON order_meta.post_id = orders.ID
+				WHERE
+					orders.post_type = 'shop_order' AND
+					orders.post_status in ( 'wc-completed', 'wc-processing', 'wc-refunded' )
+				GROUP BY orders.ID
+			) AS order_gateway
+			WHERE payment_method IS NOT NULL
+			GROUP BY payment_method
+			ORDER BY orders_count DESC
+		"
+		);
+
+		foreach ( $orders_by_payment_method as $orders_count ) {
+			$method                                  = 'gateway_' . $orders_count->payment_method;
+			$order_counts_by_payment_method[ $method ] = $orders_count->orders_count;
```

Comment:
Trivial formatting comment: the equals signs don't quite line up.

Reply:
Thanks. I missed it. 🤦‍♀️